src/main/java/com/kazuma/quizapplication/QuizJFrame.java

```diff
@@ -148,18 +153,30 @@ private void loadNextQuestion()
 			+ "Number of Correct answers = " + correctAnswers + "<br />"
```

Comment:
This section 	    
`if (questionIndex == 4)
	    {
		String output = "<html>Thanks for Taking the quiz.<br />"
			+ "Number of questions answered = " + (questionIndex + 1) + "<br />"
			+ "Number of Correct answers = " + correctAnswers + "<br />"
			+ "Number of Incorrect answers = " + wrongAnswers + "</html>";
		JOptionPane.showMessageDialog(this, output);
		initQuiz();
	    }`
probably should be in the else section on line 164.  Have one exit point

Reply:
This is a duplication